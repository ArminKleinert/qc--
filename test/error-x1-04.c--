

const 
{
    x = 2 + 3.1415;
}

section "data" 
{
    a:
    bits32[5];

}

