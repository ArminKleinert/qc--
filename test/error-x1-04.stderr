Character 19-29 Error: illegal operands
